src/components/generate_prompts/fondant_component.yaml

```diff
@@ -1,12 +1,10 @@
 name: Generate prompts
 description: Component that generates a set of seed prompts
-image: ghcr.io/ml6team/generate_prompts:0.7.0
+image: needs_to_be_build
```

Comment:
The goal was to build these in our CI/CD. I think this looks a bit strange.

Reply:
Built manually using `fondant build` for now. Still need to add a CI/CD pipeline to this repo.